content/rendering-with-batch/aws-fis/fis-start-job.md

```diff
@@ -0,0 +1,21 @@
+---
+title: "Starting a new render job"
+date: 2021-09-06T08:51:33Z
+weight: 154
+---
+
+This section of the lab will start a new rendering job. This second job will be intentionally interrupted using AWS FIS.
+
+{{% notice warning %}}
+This section has dependencies on the completion of the [AWS Batch section]({{< ref batch >}}) of the previous lab.  Please ensure you have followed all steps, the previous job has completed, and you have verified the output by following the [monitoring steps in the previous lab]({{< ref monitor.md >}})
```

Comment:
"Please, ensure that you have followed all the steps and that the previous job has completed. Also, verify the output by [...]"